cpp/src/umap/simpl_set_embed/runner.h

```diff
@@ -29,7 +29,7 @@ using namespace ML;
 
 template <int TPB_X, typename T>
 void run(const T *X, int m, int n, MLCommon::Sparse::COO<T> *coo,
-         UMAPParams *params, T *embedding,
+         UMAPParams *params, float *embedding,
```

Comment:
Why change this from T?

Reply:
Just an error, I changed that to doubles before.